Sources/Conditions/CreditCardCondition.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  CreditCardCondition.swift
+//  FormValidatorSwift
+//
+//  Created by Onur Ersel on 2016-11-02.
+//  Copyright © 2016 ustwo. All rights reserved.
+//
+
+import Foundation
+
+
+/// Credit card types that are supported by `CreditCardCondition`. There is a `regex` property that returns the regex for validating that credit card number.
+public enum CreditCardType : String {
```

Comment:
There are a lot of SwiftLint warnings and errors on this branch (e.g. not having the colon next to the name of the type/object/key as is the case on this line). Please install SwiftLint and correct it's errors. There's only one false-positive right now which is the "weak delegate violation" in the `ValidatorControl` protocol definition.

https://github.com/realm/SwiftLint